vendor/vendor.json

```diff
@@ -43,8 +43,8 @@
 			"versionExact": "v12.5.0-beta"
 		},
 		{
-			"checksumSHA1": "e8WF/zKkHC6sKeiPaP+ulB/NSq4=",
-			"path": "github.com/Azure/azure-sdk-for-go/services/containerinstance/mgmt/2017-08-01-preview/containerinstance",
```

Comment:
can we remove the old API version from the vendor directory too, since it should be unused now?